google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -112,7 +112,7 @@ public static HttpTransportOptions getDefaultHttpTransportOptions() {
     return SCOPES;
   }
 
-  protected BigQueryRpc getBigQueryRpc() {
+  protected BigQueryRpc getBigQueryRpcV2() {
```

Comment:
There aren't specific plans for specific major versions, but this aligns the rpc implementation more closely with how we are doing the GAPIC layer. The class itself (`BigQueryRpc`) has the major version in its package, so multiple instances of the same name could co-exist in google-cloud-java; however, two getter functions with the same name can't coexist, so we have to disambiguate this way.